css/style.css

```diff
@@ -369,6 +369,65 @@ body {
   color: #253858;
 }
 
+.footer {
+  background-color: #6070ff;
+  font-style: normal;
+  font-family: Poppins, sans-serif;
+  height: 779px;
+  border-radius: 100px 0 0 0;
+  padding-top: 103px;
+  padding-right: 30px;
+  padding-left: 30px;
```

Comment:
Kindly, I suggest you to use flexbox in this section, it will be able to center the elements to the container and adapt to any size of mobile device

Reply:
Hi
The footer section already displays as flex because of the class: "default-display-flex" applied to it.
 <footer class="footer headline default-display-flex">